src/wpool_queue_manager.erl

```diff
@@ -261,7 +312,7 @@ handle_cast({send_all_event_to_available_worker, Event}, State) ->
   case gb_sets:is_empty(Workers) of
     true ->
       inc_pending_tasks(),
-      {noreply, State#state{clients = queue:in({send_event, Event}, Clients)}};
+      {noreply, State#state{clients = queue:in({send_all_event, Event}, Clients)}};
```

Comment:
Line 315 is too long:       {noreply, State#state{clients = queue:in({send_all_event, Event}, Clients)}};.